pkgs/by-name/ht/htb-toolkit/package.nix

```
@@ -0,0 +1,81 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeBinaryWrapper
+, pkg-config
+, openssl
+, stdenv
+, darwin
+, coreutils
+, noto-fonts-color-emoji
+, gnome
+, libsecret
+, bash
+, openvpn
+, nerdfonts
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "htb-toolkit";
+  version = "unstable-2024-01-17";
+
+  src = fetchFromGitHub {
+    owner = "D3vil0p3r";
+    repo = "htb-toolkit";
+    rev = "54e11774ea8746ea540548082d3b25c22306b4fc";
+    hash = "sha256-QYUqdqFV9Qn+VbJTnz5hx5I0XV1nrzCoCKtRS7jBLsE=";
+  };
+
+  cargoHash = "sha256-o71/BbCTTfUDDAxAPmeWy86CmiGuRUTovxkb5hCOLCc=";
```

hash mismatch in fixed-output derivation 

```suggestion
  cargoHash = "sha256-XDE6A6EIAUbuzt8Zb/ROfDAPp0ZyN0WQ4D1gWHjRVhg=";
```